src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx

```diff
@@ -147,7 +148,8 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ project, en
   const pageTitle = `${wbsPipe(project.wbsNum)} - ${project.name}`;
 
   return (
-    <PageLayout title={pageTitle} previousPages={[{ name: 'Projects', route: routes.PROJECTS }]}>
+    <PageLayout title={pageTitle} hidePageTitle>
+      <PageBreadcrumbs currentPageTitle={pageTitle} previousPages={[{ name: 'Projects', route: routes.PROJECTS }]} />
```

Comment:
why doesnt this work

Reply:
i.e. why are we using breadcrumbs outside the page layout here